jake/__main__.py

```diff
@@ -136,10 +136,10 @@ def __handle_iq_server(application_id, response, log, config: Config):
         "All good to go! Smooth sailing for you! No policy violations reported by IQ Server")
     _exit(0)
 
-def __get_config_from_std_in(config: Config):
+def get_config_from_std_in(config):
```

Comment:
Why did you drop the __?

Reply:
No reason other than I was getting tired of fighting with getting the import statements to work in a unit test.